tests/ptf/fabric.ptf/test.py

```diff
@@ -272,12 +295,75 @@ def runTest(self):
         testutils.verify_packets(self, exp_pkt_2to1, [self.port1])
 
 
+class ArpBroadcastTest(FabricTest):
+    def runArpBroadcastTest(self, tagged_ports, untagged_ports):
+        mac_addr_mask = ":".join(["ff"] * 6)
+        vlan_id = 10
+        next_id = vlan_id
+        mcast_group_id = vlan_id
+        all_ports = set(tagged_ports + untagged_ports)
+        arp_pkt = testutils.simple_arp_packet(pktlen=76)
+        # Account for VLAN header size in total pktlen
+        vlan_arp_pkt = testutils.simple_arp_packet(vlan_vid=vlan_id, pktlen=80)
+
+        for port in tagged_ports:
+            self.set_ingress_port_vlan(port, True, vlan_id, vlan_id)
+        for port in untagged_ports:
+            self.set_ingress_port_vlan(port, False, 0, vlan_id)
+        self.add_bridging_entry(vlan_id, arp_pkt.dst, mac_addr_mask, next_id)
```

Comment:
According to [fabric design note](https://wiki.opencord.org/display/CORD/Fabric+Design+Note#FabricDesignNote-L2Broadcast)

> If the destination MAC is not learnt, there will NOT be a flow entry matching that destination MAC. It will then fallback to a lower priority entry that matches the VLAN (subnet) and point to an L2 flood group.

We should matches VLAN only
`self.add_bridging_entry(vlan_id, "00:00:00:00:00:00", "00:00:00:00:00:00", next_id)`

Reply:
Done